src/KafkaFlow/Consumers/ConsumerWorker.cs

```diff
@@ -125,6 +123,13 @@ private async Task ProcessMessageAsync(IMessageContext context, CancellationToke
         {
             try
             {
+                await this.eventHub.FireMessageConsumeStartedAsync(
+                    new MessageEventContext
+                    {
+                        DependencyResolver = this.workerDependencyResolverScope.Resolver,
```

Comment:
maybe we don't need the Resolver here, since we have it in the `IMessageContext`

Reply:
Yes, that's it :smiley:

Reply:
I ended up keeping as it was and included a constructor with only the `IMessageContext`